pkg/scheduler/framework/v1alpha1/normalize_score.go

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
We should have this file under framework/plugins/helpers because it is not really part of the framework.

Reply:
we should also have unit tests for it.